examples/timeseries_convolution_filter.py

```diff
@@ -0,0 +1,56 @@
+"""
+======================================================
+Smoothing of TimeSeries Data Using Convolution Filters
+======================================================
+
+Sometimes, when working with scientific data, we have noisy data from which
+we need to extract low-frequency components from. This example illustrates
```

Comment:
I would remove the first sentence.

Reply:
I also would like you to add some warning or note that peak finding is a complex problem that has many potential solutions and this example is just one method of many.